docs/faq.rst

```diff
@@ -0,0 +1,26 @@
+.. _installation:
+
+**************************
+Frequently Asked Questions
+**************************
+
+Usage
+-----
+
+.. _faq-usage:
+
+My simulation seems to consume excessive amounts of memory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+High memory usage can have many reasons. Both your model
+and certain simulation settings can increase the memory
+usage significantly
+
+1. Enabling ``track_rpacket: true`` will take up substantial
+   amounts of memory, in particular for higher packet counts.
+   Consider turning this feature off or only use it with a
+   small amount of packets.
+2. Both the number of shells and the number of packets
```

Comment:
This seems reduntant. Does the user have a way of knowing how many they might need?